packages/foreman/rubygem-builder/rubygem-builder.spec

```diff
@@ -77,8 +77,9 @@ cp -a .%{gem_dir}/* \
 %license %{gem_instdir}/MIT-LICENSE
 %{gem_libdir}
 %{gem_instdir}/rakelib
-%exclude %{gem_cache}
-%{gem_spec}
+%exclude %{gem_instdir}%{gem_cache}
+%exclude %{gem_instdir}%{gem_spec}
+%exclude %{gem_instdir}/builder.blurb
```

Comment:
Can you add this here -- https://github.com/theforeman/foreman-packaging/blob/rpm/develop/gem2rpm/scl.spec.erb#L10

Just add a new ignore line with `builder.blurb`

Reply:
`%exclude %{gem_instdir}/%{name}.gemspec`